packages/terra-alert/tests/wdio/alert-spec.js

```diff
@@ -1,64 +1,61 @@
-  });
+  it('displays types of Alert', () => {
+    browser.url('/#/raw/tests/terra-alert/alert/alert-type');
 
-  describe('Wrapped Content', () => {
-    before(() => { browser.url('/#/raw/tests/terra-alert/alert/alert-with-wrapped-content'); });
-    Terra.it.validatesElement();
+    Terra.validates.element('types');
   });
 
-  describe('Responsive', () => {
-    before(() => { browser.url('/#/raw/tests/terra-alert/alert/responsive-example'); });
-    Terra.it.validatesElement();
+  it('displays Alert with Wrapped Content', () => {
+    browser.url('/#/raw/tests/terra-alert/alert/alert-with-wrapped-content');
+
+    Terra.validates.element('wrapped content');
   });
 
-  describe('Titles', () => {
-    before(() => { browser.url('/#/raw/tests/terra-alert/alert/alert-title'); });
-    Terra.it.validatesElement();
+  it('displays Responsive Alert', () => {
+    browser.url('/#/raw/tests/terra-alert/alert/responsive-example');
+
+    Terra.validates.element('responsive');
   });
 
-  describe('Custom', () => {
-    before(() => { browser.url('/#/raw/tests/terra-alert/alert/custom-alert'); });
-    Terra.it.validatesElement();
+  it('displays Alert with titles', () => {
+    browser.url('/#/raw/tests/terra-alert/alert/alert-title');
+
+    Terra.validates.element('titles');
   });
 
-  describe('Action Button', () => {
-    before(() => {
-      browser.url('/#/raw/tests/terra-alert/alert/alert-action-button');
-    });
+  it('displays custom Alert', () => {
+    browser.url('/#/raw/tests/terra-alert/alert/custom-alert');
 
-    Terra.it.validatesElement();
+    Terra.validates.element('custom');
+  });
 
-    it('should be register actions', () => {
-      expect(browser.getText('#actionAlert')).to.have.string('clicked 0 times');
-      browser.click('#actionAlert button');
-      expect(browser.getText('#actionAlert')).to.have.string('clicked 1 times');
-    });
+  it('displays Alert with action button', () => {
+    browser.url('/#/raw/tests/terra-alert/alert/alert-action-button');
 
-    after(() => browser.moveToObject('#root', 0, 700));
+    Terra.validates.element('action button');
   });
 
-  describe('Dismissible', () => {
-    before(() => {
-      browser.url('/#/raw/tests/terra-alert/alert/alert-dismissible');
-    });
+  it('should register actions', () => {
+    expect(browser.getText('#actionAlert')).to.have.string('clicked 0 times');
+    browser.click('#actionAlert button');
+    expect(browser.getText('#actionAlert')).to.have.string('clicked 1 times');
+  });
 
-    Terra.it.validatesElement();
+  it('displays dismissible Alert', () => {
+    browser.url('/#/raw/tests/terra-alert/alert/alert-dismissible');
 
-    it('should dismiss', () => {
-      browser.click('#dismissibleAlert button');
-      expect(browser.getText('#dismissed')).to.equal('Alert was dismissed');
-    });
+    Terra.validates.element('dismissible');
+  });
 
-    Terra.it.validatesElement('dismissed');
+  it('should dismiss', () => {
```

Comment:
same deal here, star with a url or combine it with another it.

Reply:
actually this is probably fine 🤷‍♂️

Reply:
This would would make sense to probably be grouped by a describe block since both its are round a dismissible alert

Reply:
Updated here https://github.com/cerner/terra-core/pull/3219/commits/40445a7a5c74cf7071e1cdd13f6195a4003461f8.